src/java.base/share/classes/jdk/internal/module/ModuleLoaderMap.java

```diff
@@ -51,32 +51,31 @@ private static final class Mapper implements Function<String, ClassLoader> {
         private static final ClassLoader APP_CLASSLOADER =
                 ClassLoaders.appClassLoader();
 
-        private static final Integer PLATFORM_LOADER_INDEX = 1;
-        private static final Integer APP_LOADER_INDEX      = 2;
+        private static final String PLATFORM_LOADER_NAME = "PLATFORM";
+        private static final String APP_LOADER_NAME      = "APP";
 
         /**
-         * Map from module to a class loader index. The index is resolved to the
+         * Map from module to a class loader name. The name is resolved to the
```

Comment:
Would you mind adjusting this to "Map from module name to class loader name"? Otherwise switching this to using a string is okay.

Reply:
Done.

Reply:
@AlanBateman Can you approve the PR? Thanks